cmd/metricchecker/cmd.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/PingCAP-QE/metrics-checker/pkg/metric"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "Metrics checker",
+	Short: "For checking prometheus metrics",
+	Run: func(cmd *cobra.Command, args []string) {
+		InitConfig(configFilePath, configBase64)
+
+		address = metric.AddHTTPIfIP(address)
+		if grafanaAPIURL != "" {
+			dashboardName := "Metrics Checker"
+			grafanaAPIURL = metric.AddHTTPIfIP(grafanaAPIURL)
+			err := metric.CreateMetricsDashboard(grafanaAPIURL, dashboardName, config.MetricsToShow)
```

Comment:
How about providing a single command to generate dashboard?

Reply:
Cool.